### PR TITLE
test: send xray reports

### PR DESCRIPTION
**Pull request by chriswldenyer**
Send reports to Jira xray. Specify which Test Plan in the Jenkinsfile. 
Currently implemented for on-demand E2E tests.

### CODE REVIEW

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

File: scripts/e2e-test.sh [hunk 1]
```diff
@@ -168,6 +178,7 @@ echo "    e2e_image_tag=$e2e_image_tag"
 echo "    e2e_docker_registry=$e2e_docker_registry"
 echo "    e2e_reports_dir=$e2e_reports_dir"
 echo "    e2e_uninstall_cleanup=$e2e_uninstall_cleanup"
+echo "    test_plan=$test_plan_arg"
```
Comment (blaisedias) [inline scripts/e2e-test.sh:181]: The unofficial convention is environment variables used to control go code behaviour, are prefixed with e2e_

**Comment by chriswldenyer** [comment]
Should add Jenkins Junit reporting as well.

**Review by blaisedias — CHANGES_REQUESTED** [review]

**Pull request closed without merging**